irohad/ametsuchi/temporary_wsv.hpp

```diff
@@ -51,7 +51,7 @@ namespace iroha {
        */
       virtual bool apply(const model::Transaction &transaction,
                          std::function<bool(const model::Transaction &,
-                                            CommandExecutor &, WsvQuery &)>
+                                            WsvCommand &, WsvQuery &)>
                              function) = 0;
```

Comment:
Is this apply not same as previous one?
In now, both command executor and command validator are out of ametsuchi.
We use apply with WsvCommand instead of CommandExecutor.
Is it right?